src/platform/site-wide/sass/shame.scss

```diff
@@ -636,3 +636,92 @@ div[data-widget-type="events"] {
   }
 }
 /* ^^^ DO NOT REMOVE w/o DS REVIEW: solves !important conflict*/
+
+/* ~~~~~ DO NOT REMOVE w/o DS REVIEW: Temporary fixes to enable core.scss swap ~~~~~~ */
+.va-flex {
+  display: flex;
+}
+.va-header-logo > img {
+  height: 59px;
+  width: 264px;
+}
+
+// Grid container
+.vads-l-grid-container,
+.vads-l-grid-container--full {
+  @include outer-container($site-max-width);
+}
+
+.vads-l-grid-container {
+  @include padding(null $site-margins-mobile);
+
+  @include media($medium-screen) {
+    @include padding(null $site-margins);
+  }
+}
+
+.vads-l-grid-container--full {
+  padding: 0;
+  max-width: 100%;
+}
+
+.vads-l-row {
+  display: flex;
+  flex-wrap: wrap;
+  min-width: 100%;
+}
+
+%grid-column {
+  box-sizing: border-box;
+  min-height: 1px; // Prevent columns from collapsing when empty
+  min-width: 0; // Resize columns as expected (https://css-tricks.com/flexbox-truncated-text/)
+  position: relative;
+  width: 100%;
+}
+
+@for $i from 1 through $grid-columns {
+  // Example: vads-l-col--6
+  .vads-l-col--#{$i} {
+    /* stylelint-disable-next-line */
+    @extend %grid-column;
+  }
+}
+
+.vads-l-col {
+  /* stylelint-disable-next-line */
+  @extend %grid-column;
+  @include equal-width-flexbox-col;
+}
+
+@for $i from 1 through $grid-columns {
+  // Example: vads-l-col--12
+  .vads-l-col--#{$i} {
+    @include flexbox-col($i);
+  }
+}
+
+// Breakpoint prefix grids. All
+@each $bp_name, $bp_value in $breakpoints {
+
+  @media (min-width: $bp_value) {
+    // Provide breakpoint classes for equal-width columns
+    // Example: vads-l-lg-col
+    .#{$bp_name}\:vads-l-col {
+      @include equal-width-flexbox-col;
+    }
+
+    @for $i from 1 through $grid-columns {
+      // Example: vads-l-lg-col--12
+      .#{$bp_name}\:vads-l-col--#{$i} {
+        @include flexbox-col($i);
+      }
+    }
+  }
+}
+
+// foundation/global.scss sets display and vertical align on img tags. This is most obviously seen in the footer image
+// in content-build
+img {
+  display: inline-block;
+  vertical-align: middle;
```

Comment:
Foundation sets these on the element selector. Without these styles, the image in `.va-banner-container` right above the footer won't sit flush to the rest of the footer. Also without these, images in promo blocks will create some minor content shift in elements above them.